### DISCUSSION TITLE
[Question] It seems that the floating window doesn't take `blend=80` argument into account.

### DISCUSSION THREAD

**Discussion by nyngwang** [MunifTanjim/nui.nvim#224] [Q&A] [answered]
Hi, I'm a user of noice.nvim. If this cannot be solved from nui.nvim side then I will report this there. This is my settings:

```lua
vim.cmd([[
  hi NoiceCmdlinePopup guibg=#101010 blend=30
  hi NoiceCmdlineItems guibg=#101010 blend=80
]])
local wininfo = vim.fn.getwininfo()[1]
local offset_top = math.floor(wininfo.height/4)
local styles = {
  position = { col = '50%', row = offset_top },
  border = { style = 'shadow', padding = { 0, 0, 0, 0 } },
  size = { width = 100 },
}
require('noice').setup {
  -- unrelated key-value's are removed/skipped.
  views = {
    cmdline_popup = vim.tbl_deep_extend('force', {}, styles, {
      win_options = { winhighlight = { Normal = 'NoiceCmdlinePopup' } },
      size = { height = 2 },
    }),
    popupmenu = vim.tbl_deep_extend('force', {}, styles, {
      win_options = { winhighlight = { Normal = 'NoiceCmdlineItems' } },
      position = { row = offset_top+2 },
      size = { height = 15 },
    }),
  },
}
```

- so It seems that the `blend=` arguments are not taken into account by nui.nvim.
- the expected behavior is that the two `Normal` menus are transparent to a degree specified by `blend=`.

<img src="https://user-images.githubusercontent.com/24765272/208675118-d87f56f6-6154-4c6e-a098-f68cfcb9cd2f.png" width="500">

**Comment by MunifTanjim** [comment, marked as answer]
This is core Neovim behavior.

For setting transparency of floating windows, check `:help 'winblend'`.

You can pass this option inside the `win_options` table.

**Reply by nyngwang** [reply]
Sorry for the wrong place of my posting. I will do it correctly the next time and thanks for your explanation!

**Reply by nyngwang** [reply]
~~So those `blend=` arguments should be removed right?~~ Strangely, I seem to have transparent `Normal` background with these options before...

**update**: I just confirmed this myself. No more questions now :)
